### ISSUE TITLE
feat: New Login Options

### ISSUE THREAD

**Issue by JustinhSE** [Namit2111/bible-verse-finder#76]
- Part 2 for #71 
Adding logins for Email and GitHub

Reference firebase documentation as well as #71 for information on how to implement email login

**Comment by devxspectre** [comment]
@JustinhSE can you please assign me the issue.

**Comment by JustinhSE** [comment]
@Sourav6971 yes, but can you provide some high level overview of what you are doing ?

**Comment by devxspectre** [comment]
@JustinhSE  sure, I'll integrate the google oauth and github oauth api using axios in the frontend and store the response tokens and the response from the api. Let me know if you want more details.

**Comment by JustinhSE** [comment]
@Sourav6971 just a note, we are using firebase so the github auth is already setup and enabled. You just need to add the component for the front end for the popup to occur and document that a user signed in , in our db. (Reference prev oauth PR)

**Comment by JustinhSE** [comment]
Due to inactivity, @Sourav6971 you have been unassigned
